api/python/src/cellxgene_ontology_guide/ontology_parser.py

```diff
@@ -1,10 +1,10 @@
 import re
 from typing import Any, Dict, List, Union
 
-from artifact_download import load_artifact_by_schema
-from constants import ALL_ONTOLOGY_FILENAME, ONTOLOGY_INFO_FILENAME
 from entities import Ontology, OntologyFileType, OntologyVariant
 
+from cellxgene_ontology_guide.supported_versions import CXGSchema
+
 
 class OntologyParser:
```

Comment:
thoughts on refactoring this class to also take an ontology_name, so we can load the ontology file to read from in init and not have to request that as an input in every method? i.e. they can initialize `cl_parser = OntologyParser("5.0.0", "CL")`

We can also take care of this in a follow-up PR, if you prefer

Reply:
I'll do that as a follow up